chat-plugins/info.js

```diff
@@ -2091,6 +2091,12 @@ const commands = {
 				// Translated analysis do not support automatic redirects from a speciesid to the proper page
 				let pageid = pokemon.name.toLowerCase().replace(' ', '_');
 				this.sendReplyBox(`<a href="https://www.smogon.com/translations/${supportedLanguages[room.language]}/analyses/ou/${pageid}">${generation.toUpperCase()} ${Chat.escapeHTML(formatName)} ${pokemon.name} analysis</a>, brought to you by <a href="https://www.smogon.com">Smogon University</a>`);
+			} else if (formatId === 'ou' && generation === 'sm') {
+				let pageid = pokemon.name.toLowerCase().replace(' ', '_');
+				this.sendReplyBox(`<a href="https://www.smogon.com/dex/${generation}/pokemon/${speciesid}${(formatId ? '/' + formatId : '')}">${generation.toUpperCase()} ${Chat.escapeHTML(formatName)} ${pokemon.name} analysis</a>, brought to you by <a href="https://www.smogon.com">Smogon University</a><br />` +
```

Comment:
This is in a block that includes `if (formatId === 'ou')`, which means that inside this block, `formatId` is always `'ou'`.

Because of this, `${(formatId ? '/' + formatId : '')}` is entirely unnecessary because it can just be replaced with `/ou`.

Reply:
That's what LGTM is warning you about.

Reply:
Thanks Zarel